src/globus_sdk/services/gcs/data/user_credential.py

```diff
@@ -0,0 +1,41 @@
+from typing import Any, Dict, Optional
+
+from globus_sdk import utils
+from globus_sdk._types import UUIDLike
+
+
+class UserCredentialDocument(utils.PayloadWrapper):
+    """
+    Convenience class for constructing a UserCredential document
+    to use as the `data` parameter to `create_user_credential` and
+    `update_user_credential`
+
+    :param DATA_TYPE: Versioned document type.
+    :type DATA_TYPE: str
+
```

Comment:
We need the full parameter list in here. Even stubby descriptions are fine, but we want full doc on all parameters.

Other datatypes in this dir should have good example text for `additional_fields`.

Reply:
Ack, can't believe I forgot param docs, good catch